golang/cosmos/scripts/protocgen.sh

```diff
@@ -9,7 +9,7 @@ for dir in $proto_dirs; do
   -I third_party/proto \
   --gocosmos_out=plugins=interfacetype+grpc,\
 Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
-  --grpc-gateway_out=logtostderr=true,allow_colon_final_segments=true:. \
+  --grpc-gateway_out=logtostderr=true:. \
```

Comment:
I was curious about this change, then I found the v2 migration docs.  We are still using v1.x, so please revert this change and rerun `make proto-gen`.

To preserve what you learned, could you please instead add a comment like:

```sh
# NOTE: when migrating to grpc-gateway v2, we will need to remove the
# allow_colon_final_segments=true
# as per https://grpc-ecosystem.github.io/grpc-gateway/docs/development/grpc-gateway_v2_migration_guide/#withlastmatchwins-and-allow_colon_final_segmentstrue-is-now-default-behaviour
```

Reply:
Done and done.